tests/voq/test_voq_ipfwd.py

```diff
@@ -728,45 +744,70 @@ def test_voq_dut_lb_ping(self, duthosts, all_cfg_facts, ttl, size, version, port
                      ttl=ttl, ttl_change=0)
         check_packet(sonic_ping, ports, 'portA', 'portA', src_ip_fld='my_lb_ip', dst_ip_fld='nbr_lb', size=size,
                      ttl=ttl, ttl_change=0)
-        check_packet(sonic_ping, ports, 'portA', 'portB', src_ip_fld='my_lb_ip', dst_ip_fld='my_ip', size=size, ttl=ttl,
+
+        if ports['portA']['dut'].get_facts()['num_asic'] > 1:
```

Comment:
I think Loopback4096 is present in single ASIC linecard as well

Reply:
On single ASIC Loopback4096 is in HIDE_INTERNAL, the script doesn't try to reach those addresses.